conans/client/conf/config_installer.py

```diff
@@ -182,3 +188,21 @@ def _process_config_install_item(item):
         path_or_url, args = [item.strip() for item in rest[1:-1].rstrip().split(",", 1)]
         args = None if "none" in args.lower() else args
     return config_type, path_or_url, args
+
+
+def _handle_hooks(src_hooks_path, dst_hooks_path, output):
```

Comment:
This is only to skip the .git folders or what?

Reply:
Yes, there is a *hooks* folder in the .git folder, so I prefer to have this function to isolate the copy of the files